Sources/PackageModel/Module.swift

```diff
@@ -80,7 +81,7 @@ extension ModuleTypeProtocol {
 
 
 public protocol XcodeModuleProtocol: ModuleProtocol, ModuleTypeProtocol {
-    var fileType: String { get }
+    func fileType(forSource source: String) -> String
```

Comment:
I think this should take a `Path` object of one form or another, if that is the expectation.